src/Uno.UI/Microsoft/UI/Xaml/Controls/TabView/TabView.cs

```diff
@@ -0,0 +1,1248 @@
+
+			m_leftContentColumn = (ColumnDefinition)GetTemplateChild("LeftContentColumn");
+			m_tabColumn = (ColumnDefinition)GetTemplateChild("TabColumn");
+			m_addButtonColumn = (ColumnDefinition)GetTemplateChild("AddButtonColumn");
+			m_rightContentColumn = (ColumnDefinition)GetTemplateChild("RightContentColumn");
+
+			var containerGrid = GetTemplateChild("TabContainerGrid") as Grid;
+			if (containerGrid != null)
+			{
+				m_tabContainerGrid = containerGrid;
+				containerGrid.PointerExited += OnTabStripPointerExited;
+			}
+
+			m_shadowReceiver = (Grid)GetTemplateChild("ShadowReceiver");
+
+			ListView GetListView()
+			{
+				var listView = GetTemplateChild("TabListView") as ListView;
+				if (listView != null)
+				{
+					//TODO:MZ:Unsubscribe when appropriate
+					listView.Loaded += OnListViewLoaded;
+					listView.SelectionChanged += OnListViewSelectionChanged;
+
+					listView.DragItemsStarting += OnListViewDragItemsStarting;
+					listView.DragItemsCompleted += OnListViewDragItemsCompleted;
+					listView.DragOver += OnListViewDragOver;
+					listView.Drop += OnListViewDrop;
+
+					listView.GettingFocus += OnListViewGettingFocus;
+
+					m_listViewCanReorderItemsPropertyChangedRevoker = RegisterPropertyChanged(listView, CanReorderItemsProperty, OnListViewDraggingPropertyChanged);
+					m_listViewAllowDropPropertyChangedRevoker = RegisterPropertyChanged(listView, UIElement.AllowDropProperty, OnListViewDraggingPropertyChanged);
+				}
+				return listView;
+			}
+			m_listView = GetListView();
+
+			Button GetAddButton()
+			{
+				var addButton = GetTemplateChild("AddButton") as Button;
+				if (addButton != null)
+				{
+					// Do localization for the add button
+					if (string.IsNullOrEmpty(AutomationProperties.GetName(addButton)))
+					{
+						var addButtonName = ResourceAccessor.GetLocalizedStringResource(SR_TabViewAddButtonName);
+						AutomationProperties.SetName(addButton, addButtonName);
+					}
+
+					var toolTip = ToolTipService.GetToolTip(addButton);
+					if (toolTip == null)
+					{
+						ToolTip tooltip = new ToolTip();
+						tooltip.Content = ResourceAccessor.GetLocalizedStringResource(SR_TabViewAddButtonTooltip);
+						ToolTipService.SetToolTip(addButton, tooltip);
+					}
+
+					addButton.Click += OnAddButtonClick; //TODO:MZ:Unsusbscribe when appropriate
+				}
+				return addButton;
+			}
+			m_addButton = GetAddButton();
+
+			if (SharedHelpers.IsThemeShadowAvailable())
+			{
+				var shadowCaster = GetTemplateChild("ShadowCaster");
+				if (shadowCaster != null)
+				{
+					ThemeShadow shadow;
+					shadow.Receivers.Add(GetShadowReceiver());
+
+					double shadowDepth = (double)SharedHelpers.FindInApplicationResources(c_tabViewShadowDepthName, c_tabShadowDepth);
+
+					var currentTranslation = shadowCaster.Translation;
+					var translation = new Vector3(currentTranslation.X, currentTranslation.Y, (float)shadowDepth);
+					shadowCaster.Translation = translation;
+
+					shadowCaster.Shadow = shadow);
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6063676)